packages/cli/commands/cms/convertFields.js

```diff
@@ -3,7 +3,7 @@ const fs = require('fs');
 const { createIgnoreFilter } = require('@hubspot/local-dev-lib/ignoreRules');
 const { isAllowedExtension, getCwd } = require('@hubspot/cli-lib/path');
 const { logger } = require('@hubspot/cli-lib/logger');
-const { walk } = require('@hubspot/cli-lib/lib/walk');
+const { walk } = require('@hubspot/local-dev-lib/fs');
```

Comment:
Not sure if it matters, but there's a small change in the behavior of `walk`. It used to log a debug log and then return an empty array. Now it throws on errors in the catch block. We may need to update all of the usages of `walk` to catch the thrown error and default to an empty array so we have parity in the behavior.

Reply:
Ooh good catch!